src/components/Tabber/Tabber.tsx

```diff
@@ -72,21 +101,22 @@ function Tabber(): ReactNode {
         <div className={scss.postModesContainer}>
           <PostModes
             currentPostModeId={tabs[currentTab].postModeOnView}
-            currentTab={tabs[currentTab].account}
+            currentTab={tabs[currentTab].account as Account}
             onChangePostMode={changePostMode}
           />
           <MainComposerBase
             accountId={tabs[currentTab].account.id.toString()}
             onChange={handleContentChange}
-            postMode={currentPostMode}
+            postMode={currentPostMode ?? undefined}
             value={
-              tabs[currentTab].posts[tabs[currentTab].postModeOnView].text ?? ''
+              tabs[currentTab].posts[tabs[currentTab].postModeOnView]?.text ??
+              ''
```

Comment:
Seria desejável colocar `tabs[currentTab].posts[tabs[currentTab].postModeOnView]?.text ??` em uma variável e trocar os locais onde é chamada, seria uma sugestão, algo como: 
```suggestion
              postModeOnViewText ?? ''
```
(reveja o nome da variável)